!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  BA_MAX_STEP: 50
  BLUR_SZ: 1
  BLUR_VAR: 0.1
  DEBUG: false
  FEATURE_THRES: 10
  FEATURE_OF_PYMD: 2
  FEATURE_BLOCKS: 12
  LOOP_CLOSURE: false
  FEATURES_PER_CELL: 1
  BA_REPROJ_DIST: 1.0
  KF_DIST: 0.1
  PROP_MIN_POINTS: 20
  BA_INLIER_THRES: 0.7
  REFINE_PIXEL: false
  PROP_PTS_RATIO: 0.5
  PROP_MAX_STEP: 50
  PROP_PYMD: 2
  PROP_PROJ_DIST: 2.0
  SCALE_MAX_STEP: 50
  SCALE_PYMD: 2
  TEST_STEREO: false
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      BA_MAX_STEP: 50
      BLUR_SZ: 1
      BLUR_VAR: 0.1
      DEBUG: false
      FEATURE_THRES: 10
      FEATURE_OF_PYMD: 2
      FEATURE_BLOCKS: 12
      LOOP_CLOSURE: false
      FEATURES_PER_CELL: 1
      BA_REPROJ_DIST: 1.0
      KF_DIST: 0.1
      REFINE_PIXEL: false
      PROP_MIN_POINTS: 20
      BA_INLIER_THRES: 0.7
      PROP_PTS_RATIO: 0.5
      PROP_MAX_STEP: 50
      PROP_PYMD: 2
      PROP_PROJ_DIST: 2.0
      SCALE_MAX_STEP: 50
      SCALE_PYMD: 2
      TEST_STEREO: false
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      name: Default
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      state: true
      type: ''
    state: []
state: []
